pages/instructorAssessments/instructorAssessments.ejs

```diff
@@ -65,6 +66,7 @@
                   <% } %></a>
                   <%- include('../partials/issueBadge', {count: row.open_issue_count}); %>
                 </td>
+                <td><%= row.assessment_unit_name %></td>
```

Comment:
Would it make sense to group it by unit for instructors as well, so instructors see them in the same order as students?

Reply:
Makes sense. Added. Do you think I should remove this column from this page?

Reply:
Maybe show the unit if sorting by set, or the set if sorting by unit?

Reply:
The set is already indicated by the badge next to the assessment name, and if groupBy is set to "set", then knowing the unit is pretty much pointless knowledge

Reply:
Good point.

Reply:
I think I'll go ahead and remove it. If people want it, it's not that hard to add back in